modules/logging/aws-for-fluent-bit/main.tf

```diff
@@ -0,0 +1,82 @@
+# Create namespace logging
+resource "kubernetes_namespace" "logging" {
+  depends_on = [
+    var.module_depends_on
+  ]
+  metadata {
+    name = var.namespace_name
+  }
+}
+
+resource "helm_release" "aws-for-fluent-bit" {
+  depends_on = [
+    var.module_depends_on
+  ]
+
+  name       = "aws-for-fluent-bit"
+  repository = "https://aws.github.io/eks-charts"
+  chart      = "aws-for-fluent-bit"
+  version    = "0.1.3"
+  namespace  = var.namespace_name
```

Comment:
you create namespace but set namespace by variable?

Reply:
Fixed